### PR TITLE
added UserController

### PR DESCRIPTION
**Pull request by entertainment-veks**
added findByUser in UserDao and UserService
added config files

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -34,10 +31,37 @@
...
+            <groupId>javax.servlet.jsp</groupId>
+            <artifactId>javax.servlet.jsp-api</artifactId>
+            <version>2.3.3</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>javax.servlet.jsp.jstl</groupId>
+            <artifactId>javax.servlet.jsp.jstl-api</artifactId>
+            <version>1.2.2</version>
+        </dependency>
```
Comment (Fedich09) [inline pom.xml:50]:
```suggestion

```

File: src/main/java/spring/intro/controller/UserController.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import spring.intro.dto.UserResponseDto;
+import spring.intro.model.User;
+import spring.intro.service.UserMapper;
+import spring.intro.service.UserService;
+
+@RestController
+@RequestMapping("/user")
+@ComponentScan(basePackages = {"spring.intro.controller"})
+public class UserController {
+
```
Comment (Fedich09) [inline src/main/java/spring/intro/controller/UserController.java:20]:
```suggestion
```

**Review by Fedich09 — CHANGES_REQUESTED** [review]

**Review by Fedich09 — APPROVED** [review]

File: sample/servlet/container/deploy/directory/META-INF/MANIFEST.MF [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Manifest-Version: 1.0
```
Comment (dmytro-hryhoruk) [inline sample/servlet/container/deploy/directory/META-INF/MANIFEST.MF:1]: ?

File: src/main/java/spring/intro/controller/UserController.java [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import spring.intro.dto.UserResponseDto;
+import spring.intro.model.User;
+import spring.intro.service.UserMapper;
+import spring.intro.service.UserService;
+
+@RestController
+@RequestMapping("/user")
+@ComponentScan(basePackages = {"spring.intro.controller"})
```
Comment (dmytro-hryhoruk) [inline src/main/java/spring/intro/controller/UserController.java:18]: it's not a configuration bean

File: src/main/java/spring/intro/dto/UserResponseDto.java [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+    public String getLogin() {
+        return login;
+    }
+
+    public void setLogin(String login) {
+        this.login = login;
+    }
+}
+
```
Comment (dmytro-hryhoruk) [inline src/main/java/spring/intro/dto/UserResponseDto.java:23]:
```suggestion
```

**Review by dmytro-hryhoruk — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (dmytro-hryhoruk) [reply pom.xml:50]: please remove this dependency

File: pom.xml [hunk 6]
```diff
@@ -34,10 +31,37 @@
...
+            <artifactId>servlet-api</artifactId>
+            <version>2.5</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>javax.servlet.jsp</groupId>
+            <artifactId>javax.servlet.jsp-api</artifactId>
+            <version>2.3.3</version>
+            <scope>provided</scope>
+        </dependency>
```
Comment (dmytro-hryhoruk) [inline pom.xml:45]: this too

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**